lib/trino-parquet/src/test/java/io/trino/parquet/ParquetTestUtils.java

```diff
@@ -121,6 +122,19 @@ public static ParquetReader createParquetReader(
             List<String> columnNames,
             TupleDomain<String> predicate)
             throws IOException
+    {
+        return createParquetReader(input, parquetMetadata, new ParquetReaderOptions(), memoryContext, types, columnNames, predicate);
+    }
+
+    public static ParquetReader createParquetReader(
```

Comment:
Don't add another method, just update existing usages to add `new ParquetReaderOptions()`